iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -454,13 +502,12 @@ private StorageDescriptor storageDescriptor(TableMetadata metadata, boolean hive
     return storageDescriptor;
   }
 
-  @VisibleForTesting
-  HiveCommitLock createLock() throws UnknownHostException, TException, InterruptedException {
-    return new HiveCommitLock(conf, metaClients, catalogName, database, tableName);
+  HiveLock lockObject() {
```

Comment:
wasn't it replaced with `HiveLock lockObject(TableMetadata metadata)`

Reply:
Thank you for spotting it. 

Fixed.